### ISSUE TITLE
Fails to build with Shapely 2.0

### ISSUE THREAD

**Issue by sebastic** [dicompyler/dicompyler-core#336]
`test_structure_volume_holes` fails:
```
======================================================================
ERROR: test_structure_volume_holes (tests.test_dicomparser.TestRTStructureSet)
Test if a structure volume with holes can be calculated.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/python-dicompylercore-0.5.5/tests/test_dicomparser.py", line 219, in test_structure_volume_holes
    self.dp.CalculateStructureVolume(coords, 3), volume)
  File "/build/python-dicompylercore-0.5.5/dicompylercore/dicomparser.py", line 628, in CalculateStructureVolume
    ordered_polygons = sorted(polygons, key=Within, reverse=True)
TypeError: '<' not supported between instances of 'Polygon' and 'Polygon'

----------------------------------------------------------------------
Ran 56 tests in 5.526s

FAILED (errors=1)
Test failed: <unittest.runner.TextTestResult run=56 errors=1 failures=0>
Structure:            test              test          Rel Diff       Abs diff
-----
DVH Type:  cumulative, abs dose: Gy, abs volume: cm3
Volume:            12.81 cm3         12.81 cm3         +0.00 %          +0.00
Max:               14.58 Gy          14.58 Gy          +0.00 %          +0.00
Min:               14.07 Gy          14.07 Gy          +0.00 %          +0.00
Mean:              14.29 Gy          14.29 Gy          +0.00 %          +0.00
D100:              14.06 Gy          14.06 Gy          +0.00 %          +0.00
D98:               14.12 Gy          14.12 Gy          +0.00 %          +0.00
D95:               14.14 Gy          14.14 Gy          +0.00 %          +0.00
D2cc:              14.39 Gy          14.39 Gy          +0.00 %          +0.00
Structure:            test              test          Rel Diff       Abs diff
-----
DVH Type:  cumulative, rel dose, abs volume: cm3
Volume:            12.81 cm3         12.81 cm3         +0.00 %          +0.00
Max:              104.14 %          104.14 %           +0.00 %          +0.00
Min:              100.50 %          100.50 %           +0.00 %          +0.00
Mean:             102.04 %          102.04 %           +0.00 %          +0.00
D100:             100.43 %          100.43 %           +0.00 %          +0.00
D98:              100.86 %          100.86 %           +0.00 %          +0.00
D95:              101.00 %          101.00 %           +0.00 %          +0.00
V100:              12.81 %           12.81 %           +0.00 %          +0.00
V95:               12.81 %           12.81 %           +0.00 %          +0.00
V5:                12.81 %           12.81 %           +0.00 %          +0.00
D2cc:             102.79 %          102.79 %           +0.00 %          +0.00
Structure: test
-----
DVH Type:  cumulative, abs dose: Gy, abs volume: cm3
Volume:    12.81 cm3
Max Dose:  14.58 Gy
Min Dose:  14.07 Gy
Mean Dose: 14.29 Gy
D100:      14.06 Gy
D98:       14.12 Gy
D95:       14.14 Gy
D2cc:      14.39 Gy
Structure: test
-----
DVH Type:  cumulative, rel dose, abs volume: cm3
Volume:    12.81 cm3
Max Dose:  104.14 %
Min Dose:  100.50 %
Mean Dose: 102.04 %
D100:      100.43 %
D98:       100.86 %
D95:       101.00 %
V100:      12.81 cm3
V95:       12.81 cm3
V5:        12.81 cm3
D2cc:      102.79 %
```

**Comment by sebastic** [comment]
Shapely 2.0 has been released:

 https://github.com/shapely/shapely/releases/tag/2.0.0

**Comment by tillea** [comment]
Any progress with this issue?

**Comment by bastula** [comment]
Not yet, will be taking a look this or next week. I don't believe it will be a difficult fix.

**Comment by bastula** [comment]
Fixed by [b399f5f](https://github.com/dicompyler/dicompyler-core/commit/b399f5f59d5d59a08dbd6b1b4329ca51e2e86624).

**Comment by tillea** [comment]
Would you mind tagging a new release including this fix?
